test/e2e/app-dir/ppr-navigations/prefetch-navigation/app/catch-all/[[...slug]]/page.tsx

```diff
@@ -10,6 +11,14 @@ export default async function Page({ params }) {
           <Link href={`/catch-all/${i}`}>Go to /catch-all/{i}</Link>
         </div>
       ))}
+      <Suspense fallback={<div id="fallback">Loading...</div>}>
+        <Dynamic />
+      </Suspense>
     </div>
   )
 }
+
+function Dynamic() {
+  unstable_noStore()
+  return <div id="dynamic">Dynamic content</div>
+}
```

Comment:
I had to add this to the test because the initial assumption was wrong: the test expects a dynamic request when navigating, but since the route could be fully statically prefetched, there was no reason for a dynamic request until I added this part.